buildbot/dependency.conf

```diff
@@ -1,31 +1,33 @@
 [VERSIONS]
-# https://github.com/intel/llvm/releases/download/2020-WW20/oclcpuexp-2020.10.6.0.4_rel.tar.gz
-ocl_cpu_rt_ver=2020.10.6.0.04
-# https://github.com/intel/llvm/releases/download/2020-WW20/win-oclcpuexp-2020.10.6.0.4_rel.zip
-ocl_cpu_rt_ver_win=2020.10.6.0.04
+# https://github.com/intel/llvm/releases/download/2020-WW31/oclcpuexp-2020.10.7.0.15_rel.tar.gz
+ocl_cpu_rt_ver=2020.10.7.0.15
+# https://github.com/intel/llvm/releases/download/2020-WW31/win-oclcpuexp-2020.10.7.0.15_rel.zip
+ocl_cpu_rt_ver_win=2020.10.7.0.15
 # Same GPU driver supports Level Zero and OpenCL:
-# https://github.com/intel/compute-runtime/releases/tag/20.21.16886
-ocl_gpu_rt_ver=20.21.16886
+# https://github.com/intel/compute-runtime/releases/tag/20.25.17111
+ocl_gpu_rt_ver=20.25.17111
```

Comment:
Please use the following gpu rt.
#https://github.com/intel/compute-runtime/releases/tag/20.29.17408
ocl_gpu_rt_ver=20.29.17408